tests/dragonfly/redis_replicaiton_test.py

```diff
@@ -68,7 +68,11 @@ async def check_data(seeder, replicas, c_replicas):
 @pytest.fixture(scope="function")
 def redis_server() -> RedisServer:
     s = RedisServer()
-    s.start()
+    try:
+        s.start()
+    except FileNotFoundError as e:
```

Comment:
Why?

Reply:
I would like to separate between a test failure and the incorrect test environment setup. In this case, it's more convenient for a developer to decide whether he wants to install the missing pieces, but at least the test pipeline does not stop. one will still get the summary at the end which tests are skipped.

Reply:
I think it will be skipped silently in CI without anyone notices, we also apply this anywhere else.
IMHO, it is best for root cause to propagate.
I guess it annoyed you when invoking locally all tests and you don't have the expected redis binary?
I can think of how to make it more friendly for that use case.